src/webapp/components/about/About.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+import { Info as InfoIcon } from "@material-ui/icons";
+import { Link } from "react-router-dom";
+
+interface AboutProps {
+    visible: boolean;
+}
+
+export const About: React.FC<AboutProps> = React.memo(({ visible }) => {
+    return (
+        <>
+            {visible && (
```

Comment:
No need to change, but I think the visible prop is no needed, right? Since this component is only being used in the router page and the `visible` prop is always true.

It could be `<About />`  instead `<About visible={true} />`

In case we keep the `visible` prop to make it reusable in the future it could be:

```tsx

if (!visible) return null;

return (
  <AboutButton to="/about">
    <InfoIcon fontSize="small" />
  </AboutButton>
)

```

Reply:
Indeed, I'll spare the prop. Thanks!